daemon/networkdriver/bridge/driver.go

```diff
@@ -230,17 +233,25 @@ func InitDriver(job *engine.Job) error {
 		}
 	}
 
+	if hairpinMode {
+		// Enable loopback adresses routing
+		sysPath := filepath.Join("/proc/sys/net/ipv4/conf", bridgeIface, "route_localnet")
+		if err := ioutil.WriteFile(sysPath, []byte{'1', '\n'}, 0644); err != nil {
+			logrus.Warnf("WARNING: unable to enable local routing for hairpin mode: %s\n", err)
```

Comment:
Good question :-/ It's the same kind of things we have [here](https://github.com/docker/docker/blob/06433cf812e26e468b9dcb6b0817f6597f1c397c/daemon/networkdriver/bridge/driver.go#L219), [here](https://github.com/docker/docker/blob/06433cf812e26e468b9dcb6b0817f6597f1c397c/daemon/networkdriver/bridge/driver.go#L225), or [here](https://github.com/docker/docker/blob/06433cf812e26e468b9dcb6b0817f6597f1c397c/daemon/networkdriver/bridge/driver.go#L228)... I'd rather keep it a warning for consistency, and maybe review the alert level of all in a followup PR.